### PR TITLE
Convert js monitor from CommonJS to ES6 modules

### LINKED ISSUE #4178: Convert JS monitor from CommonJS to ES6 modules
### Problem

Refer to https://github.com/hashgraph/hedera-mirror-node/issues/2874

### Solution

Convert the JS monitor module from CommonJS to ES6 modules

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by xin-hedera**
**Description**:

This PR converts the js monitor from CommonJS to ES6 modules

**Related issue(s)**:

Fixes #4178 

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by edwin-greene — DISMISSED** [review]
Looks good.

File: hedera-mirror-rest/monitoring/monitor_apis/package.json [hunk 1]
```diff
@@ -2,14 +2,15 @@
...
   "engines": {
     "node": ">=16.13.0 < 17"
   },
   "scripts": {
-    "test": "jest --watchAll",
-    "dev": "nodemon app.js"
+    "dev": "nodemon --experimental-specifier-resolution=node server.js",
+    "start": "node --experimental-specifier-resolution=node server.js"
   },
   "author": "Hedera Mirror Node Team",
```
Comment (steven-sheehy) [inline hedera-mirror-rest/monitoring/monitor_apis/package.json:15]: Can we manually update some of the previously rejected es6 only packages?

Comment (xin-hedera) [reply hedera-mirror-rest/monitoring/monitor_apis/package.json:15]: done

**Review by steven-sheehy — APPROVED** [review]
LGTM

**Review by edwin-greene — APPROVED** [review]

**Pull request merged**